LoadTests/SET_UP_TEST_ENTITIES.sql

```diff
@@ -1,26 +1,31 @@
 DO $$
 
     DECLARE
+        -- A high starting number to ensure that the IDs don't interfere with any existing data
         STARTING_ID INTEGER := 300000000;
         NUM_OF_USERS INTEGER := 20000;
         INDEX INTEGER := 1;

```

Comment:
Yeah I completely agree, it threw me at first - it was like this originally though so I haven't actually changed how this works! Didn't want to mess with it really since so many things (e.g. users_organisations.csv) rely on these numbers 😅